src/GlassCat/GlassTypes.jl

```diff
@@ -338,170 +338,3 @@ function info(io::IO, glass::Glass)
 end
 
 info(g::AbstractGlass) = info(stdout, g)
-
```

Comment:
is the docstring function being permanently deleted or did you just move it somewhere? Even though we're not using it much now it seems to be functioning ok. Other people might find it useful for making tools to analyze glasses. Is it necessary to remove it?

Reply:
I think I took it out because it seemed like duplicate code (I can't remember where from), it was unused, and it was lowering our test coverage.

Reply:
> Just had a look and it seems to be essentially the same as info() above but without the padding. I'll add it back in with a third function to handle with the shared aspects.

This is basically what it did - just printed the same stuff as info with markdown formatting for use in docstrings

Reply:
@BrianGun you can have the final say - if you want to keep the code then feel free to `git revert 
d3197a2`